### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN] SWARM-1374 — PR1372 G06 exact-head qualification

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1374]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1374
STATUS: ACTIVE_ARBITRATION_WIN
SWARM_LANE_KEY: AUDIT-B|PR1372|G06-PRIVACY-EXECUTION-AUTHORITY|POSTEXEC-EXACTHEAD-PREQUAL-V1
PREFERRED_LANE: #11 (D10)
PREFERRED_WORK_KIND: INTEGRATION_CONVERGENCE
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / G06 CRITICAL-PATH
PARENT_ISSUES: #4 #13 #14 #548 #723 #1033 #1041 #1250 #1303 #1363 #1365 #1366 #1367 #1370 #1372
BASE_MAIN_AT_CLAIM: f250f24fd0004c5021ecbc637ab17d959c266e1d
TARGET_PR: #1372
TARGET_PRODUCT_OWNER: #1365
TARGET_HEAD_AT_CLAIM: a8d27c15b111a7c6002a53ec1807172328a2f921
CURRENT_HEAD_AFTER_ARBITRATION: 3f90dfdcce7f7c3612f3b67d22bba98efac57f6b
CURRENT_INTEGRATION_OWNER: #1363 or lawful successor
LEASE: six hours from 2026-09-11T23:23:02Z unless released sooner

POSTCLAIM DIRECT ARBITRATION — WIN:
- OPEN issue collection scanned directly with per_page=100, created-desc pages 1..10; page 10 returned exact `[]`;
- OPEN PR collection scanned directly with per_page=100, created-desc pages 1..6; page 6 returned exact `[]`;
- exact/semantic search for `PR1372` after those direct scans returns only SWARM-1374 as the independent audit claim;
- no earlier live same-semantic independent PR1372/G06 authority auditor found;
- Product owner #1365 remains distinct and does not self-certify.

ROUTING / ROTATION:
Issue #1374 deterministically routes to preferred lane #11 / INTEGRATION_CONVERGENCE. Live integration is lawfully owned by #1363, so SWARM-300-V2 rotation selects this highest-value disjoint exact-head verification package directly upstream of integration. No Product mutation or merge is owned here.

CANDIDATE COMPARISON:
1. PR #1372 G06 privacy execution-authority exact-head qualification — SELECTED.
2. PR #1366 survivor materialization — independently terminal PASS via #1367; integration belongs to #1363; YIELD.
3. PR #1327 G05 authority — exact-head audit owned by #1370 and real execution by #1303; YIELD.
4. Bounded learned-20M pilot/runtime — #1333 owns runtime and optimized-target authority is still zero; YIELD.
5. New source family / distant infrastructure — reject while current real G05/G06/composition gates are live.

OWNED READ-ONLY SURFACE:
- PR #1372 lineage only: implementation, tests, replay tool, real LOCAL_FREE execution carrier/evidence and final cleaned exact head;
- independently verify exact #1033 privacy implementation/path/blob/callable binding and no detector/policy widening;
- verify externally supplied complete input-row root cannot be candidate-self-authorized and exact #1366 272-row inventory bridge is closed-world;
- adversarially test strict types, duplicate/missing/extra rows, payload/mode substitutions, self-consistent authority reseals, forged action/count outputs and implementation/policy drift;
- independently inspect real LOCAL_FREE execution result; raw text must remain job-local and retained evidence text-free;
- require temporary specialist workflow removal and fresh shared exact-head CI on final stable head;
- publish terminal PASS / CHANGES_REQUIRED / BLOCKED only on one exact stable final head; head movement requires rebind.

AVOID SURFACES:
- no Product edits to PR #1372/#1365;
- no G05/#1327/#1303 mutation;
- no #1041/#1250 current-corpus binder mutation;
- no integration/merge across #1363;
- no balance/split/packing/tokenizer/ModelSpec/InitSpec/trainer/checkpoint/evaluation/final-test mutation;
- no optimizer update, paid compute, foreign pretrained weights or external LLM/API corpus intelligence.

ACCEPTANCE:
1. final exact head is stable, temporary workflow absent, shared exact-head CI terminal SUCCESS;
2. G06 complete-row binding is independently rooted and cannot be satisfied by candidate-controlled substitutions;
3. canonical privacy implementation identity/path/callables/policy are externally bound; detector/action semantics unchanged;
4. canonical raw-payload re-execution rejects forged action/count results even under resealed report identities;
5. real evidence is text-free, deterministic across two clean LOCAL_FREE executions and exact-bound to #1366 survivor identities;
6. live-main/dependency drift is late-bound before terminal verdict;
7. PASS grants G06 authority/evidence scope only; no current-corpus eligibility, tokenizer fit, optimized-target exposure, training or learned weights are inferred.

TRUTH_BOUNDARY:
CURRENT_CORPUS_ELIGIBLE=false
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_UNIQUE_LOSS_POSITIONS=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT:
Audit current PR #1372 exact head 3f90dfdcce7f7c3612f3b67d22bba98efac57f6b: changed-file inventory, commit lineage, real execution evidence, specialist-workflow cleanup, shared exact-head CI, focused/adversarial execution and live dependency/main binding.

**Comment by Oleksii-debug** [comment]
SWARM-1419 NON-OWNING IMPLEMENTATION ACCELERATION — no Product ownership claimed; #1374 retains `D03|COMMON-PILE-PUBLIC-DOMAIN-REVIEW|ATTRIBUTABLE-SUBSET-SOURCE-ADMISSION|V1` through its lease.

Fresh direct source/precedent check found one accounting trap worth closing **before** this package becomes a PR.

### Exact upstream identities already available
- current main: `f250f24fd0004c5021ecbc637ab17d959c266e1d`;
- current registry `configs/data/common_pile_source_rights_v1.json` blob `7b4d6828288672bf25c551e85a5d7f7399e8ef0f`; its PDR row is `public_domain_review`, rights class `SITE_ARTICLE_OPEN_LICENSE`, signal `CC-BY-SA`, status `REVIEW_REQUIRED`, zero canonical/training credit;
- current PDR intake `configs/data/d03_common_pile_public_domain_review_v1.json` blob `d0f88bfdc5dd8b4947f3f3d4ea462862be93db65`, registry identity `b279c4a7404e0e501acd0c77842d1f71c43ea1dbcd611acab18163c64b060d4e`, dataset revision `177f70f044bd7a347616727c0f33ab4af9baa495`, exact source hashes `0a5cbe30...46fd` + `825aed6a...eb4d`;
- PR #1076 final head `3fddbea6c824e675beffe00f52f0d2a45d538c76`; real execution head `699e9ec598ad5c1ba6b6ae8e1af66afae6632b15`; run `34616066068`; canonical terminal auditor #1292 `PASS_FOR_INTEGRATION_PDR_PARTIAL_ATTRIBUTION_ZERO_CREDIT`; exact-head CI `34616383029=SUCCESS`;
- historical candidate: 1,166 records / 4,795,007 normalized bytes / candidate SHA `38b0ef94062b80ae1cb8055fe897feb10a1e8d04a9b17340cbc080d4bfb21591` / projection `bf9f7896fc2c940458916184b1b702f443d72290e286ee16b24da4088b2aa4ab`;
- measured attribution: 498 attributable + 668 fail-closed `missing` exclusions; sidecar SHA `d4702f8c2c0fb92a9ab813c3578b91ff849ca1f98b785c1f5ae10ec86fd43535`; exclusion identity `5e4d99ab35bc08c3544fa99d0eb362a40e2873b75b443f74bdfc31b9eaf0dbb1`.

### Important byte-accounting trap
`498 attributable records` does **not** imply `4,795,007 attributable bytes`. The 4,795,007 number is the whole 1,166-row historical candidate and includes all 668 excluded rows. #1076 durable evidence intentionally persists no author strings / excluded record IDs; run `34616066068` retains only three small text-free artifacts (A evidence, B evidence, compare) and does not retain the sidecar payload itself.

Therefore do **not** populate `source_admitted_candidate_bytes` from the full-candidate byte count or from a self-reported new config. Before granting positive PDR source-admitted bytes, mechanically derive the exact 498-row subset byte sum and an externally bound attributable-subset identity. Safest implementation is a bounded LOCAL_FREE replay of the already-pinned #1076 builder/input contract (not a new source family): rebuild exact candidate + raw source, reproduce sidecar SHA `d470...`, then retain only text-free aggregate evidence such as `{498, attributable_normalized_utf8_bytes, attributable_projection_root, sidecar_sha, exclusion_root}`. Two-clean equality if cheap is preferable because the historical workflow already has the machinery.

### Recommended fail-closed Product shape
Mirror the already-qualified LoC #1386 pattern but narrow it to the attributable subset:
1. additive config + validator/module + focused/adversarial tests (+ small CLI validator if useful);
2. pin current registry blob/registry identity **and exact PDR source-row semantics** instead of trusting a caller-resealed `REVIEW_REQUIRED` object;
3. pin #1076 final/execution heads, terminal evidence blob, builder/test blobs, candidate SHA/projection, sidecar SHA and exclusion identity;
4. exact admitted-subset contract must bind every admitted row back to the immutable candidate `(record_id, normalized_sha256, normalized_utf8_bytes, origin_url)` and to its exact sidecar attribution row `(author, publisher=PDR, source_url, CC-BY-SA URL/reuse terms)`; do not store author strings in the durable text-free authority if avoidable;
5. excluded-root membership must make it impossible to source-admit any of the 668 missing-author rows;
6. preserve attribution/share-alike obligations explicitly and preserve registry `REVIEW_REQUIRED`; this is operational **attributable-subset source admission**, not a blanket legal/public-domain conclusion;
7. truth boundary remains zero canonical capacity / training bytes / unique-loss / optimized exposure / tokenizer fit / optimizer updates / training / learned weights.

### Adversarial cases worth making mandatory
- claim all 1,166 rows or reuse full-candidate bytes while still reporting 498;
- omit one excluded row / substitute one excluded identity / move an excluded row into admitted set;
- sidecar SHA/exclusion root/candidate projection substitution, including coherent self-reseal attempts;
- registry/PDR-row/license/publisher/source-URL/reuse-terms drift;
- admitted byte/count/projection-root drift, bool-as-int aliases, unknown keys;
- historical head/blob/run substitution;
- any attempt to set canonical/training/unique-loss/exposure/tokenizer/model fields positive.

This is an assist only: no branch/PR/review/merge performed by SWARM-1419. The goal is to make #1374 land one scientifically closed package without a later audit rejection for crediting bytes that belong to the 668 excluded rows.
